tasks/src/text-to-video/about.md

```diff
@@ -0,0 +1,115 @@
+
+
+### Pose-Control
+
+Generating text-editable and pose-controllable character videos have a rising demand in creating digital humans/avatars. By utilizing easily obtained datasets (i.e.,image pose pair and pose-free video) and the pre-trained text-to-image (T2I) model to obtain the pose-controllable character videos, this learn a zero-initialized convolutional encoder to encode the pose information and then finetune the motion of the network via a pose-free video dataset.
+
+
+
+## Task Variants
+
+There are different variants based on the inputs and outputs:
+
+### Text to Video Retrieval
+
+This is a task that involves finding videos that are relevant to a given text query. This task is challenging because it requires the model to understand the meaning of the text query and to match it to the visual content of the videos. The best approach to text-to-video retrieval depends on the specific application. For example, if the application is to find videos that are relevant to a given text query, then a text-based model may be the best approach. However, if the application is to find text queries that are relevant to a given video, then a video-based model may be the best approach.
+
+
+### Text-to-Video Editing
+
+One T2V model variant is text-based video style and local attribute editing from the trained text-to-image model. Text to video editing models can make it easier to perform tasks like cropping, stabilization, color correction,resising and audio editing etc.,  and they can also help to ensure that the tasks are performed consistently
+
+
+### Text to Video Search
+
+Text to video search is a task of retrieving videos that are relevant to a given text query. This can be challenging, as videos are a complex medium that can contain a lot of information. By using semantic analysis to extract the meaning of the text query, visual analysis to extract features from the videos, such as the objects and actions that are present in the video, and temporal analysis to categorize  relationships between the objects and actions in the video, we can determine which videos are most likely to be relevant to the text query.
+
+### Text-driven Video Prediction
+
+Video Prediction is the task of predicting future frames given past video frames. Text-driven video prediction (TVP) is a task that involves generating a video sequence from a text description. The text description can be anything from a simple sentence to a detailed story. The goal of TVP is to generate a video that is both visually realistic and semantically consistent with the text description.
+
+
+### Video translation: 
+
+Text-to-video models can be used to translate videos from one language to another or allow to query the multilingual text-video model with non-English sentences. This can be useful for people who want to watch videos in a language that they don't understand, especailly when multi-lingual captions are available for training.

```

Comment:
```suggestion
Text-to-video translation models can translate videos from one language to another or allow to query the multilingual text-video model with non-English sentences. This can be useful for people who want to watch videos in a language that they don't understand, especially when multi-lingual captions are available for training.

```

Interesting, how is the input text and output video here? Can you explain?

Reply:
There is a latent embedding space where the text input in English is converted to a representation that translates to concepts used to generate videos. If we are training a model to generate video from an English caption, we could expand the dataset to have multiple language captions per video to form a larger vocabulary that has artificial code switching(replacing some random repeating words into other language vocab) and this helps borrows statistical properties and distributions from higher resource language like English , to some lower resource language where data or text annotation isn't available, let's say Turkish. Obviously all concepts cannot be expressed in English and there are words in other languages like French or German to describe situations or objects. Searching or prompting for a video in any language should be able to pull up or generate a similar video about the same concept

Reply:
There is a latent embedding space where the text input in English is converted to a representation that translates to concepts used to generate videos. If we are training a model to generate video from an English caption, we could expand the dataset to have multiple language captions per video to form a larger vocabulary that has artificial code switching(replacing some random repeating words into other language vocab) and this helps borrows statistical properties and distributions from higher resource language like English , to some lower resource language where data or text annotation isn't available, let's say Turkish. Obviously all concepts cannot be expressed in English and there are words in other languages like French or German to describe situations or objects. Searching or prompting for a video in any language should be able to pull up or generate a similar video about the same concept 

I was brainstorming use-cases . We can remove this if it doesn't align completely or sounds ambiguous.